<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
					background-image: url(:/LoginWindow.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="signUpWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>510</width>
      <height>550</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="LE_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>491</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_username">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>491</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_signUp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-family: 'Fjalla One', sans-serif;
font-size: 40px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_signUp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>450</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color:#4f68ff;
	border-radius:29px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:17px;
	font-weight:bold;
	padding:21px 45px;
	text-decoration:none;
	text-shadow:0px 1px 0px #2f6627;
}

QPushButton:hover {
	background-color:#687eff;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_email">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>491</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 11px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: none;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;</string>
        </property>
        <property name="text">
         <string>Already have an account? </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_signUp_Login">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
	font-weight : bold;

}
QPushButton:hover{
	change-cursor: cursor('PointingHand');

}</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>491</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="LE_name">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_lastname">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="LB_errorName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>165</y>
       <width>241</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorEmail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>489</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorUsernameSnUp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>489</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorPassSnUp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>489</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorLastname">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>165</y>
       <width>241</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>510</width>
      <height>550</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 11px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: none;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;</string>
        </property>
        <property name="text">
         <string>Don't have an account? </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_login_signUp">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
	font-weight : bold;

}
QPushButton:hover{
	change-cursor: cursor('PointingHand');

}</string>
        </property>
        <property name="text">
         <string>Sign up</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-family: 'Fjalla One', sans-serif;
font-size: 40px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_login">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>450</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color:#4f68ff;
	border-radius:29px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:17px;
	font-weight:bold;
	padding:21px 45px;
	text-decoration:none;
	text-shadow:0px 1px 0px #2f6627;
}

QPushButton:hover {
	background-color:#687eff;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="LB_pass">
        <property name="styleSheet">
         <string notr="true">font-family: Arial;
font-size: 20px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_loginPass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>140</y>
       <width>491</width>
       <height>116</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="LB_username">
        <property name="styleSheet">
         <string notr="true">font-family: Arial;
font-size: 20px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_loginUser">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	background-color:#F1EEF4;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size:17px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="LB_errorUsername">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>489</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>489</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Calibri;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ec5d8a;
font-weight: bold;
font-style: normal;
text-transform: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
